examples/deposit.ts

```diff
@@ -1,43 +1,53 @@
-import { User } from "../libs/user";
+import {
+  UserFactory,
+  BLOCKCHAIN_WS_URL_DEFAULT,
+  CLIENT_API_URL_DEFAULT,
+} from "../libs/user";
 import * as dotenv from "dotenv";
 import path from "path";
 
-const _rootPath = path.resolve();
-dotenv.config({ path: path.join(_rootPath, ".env") });
+const rootPath = path.resolve();
+dotenv.config({ path: path.join(rootPath, ".env") });
 
 // Script config for ganache
-// const ethereumPrivateKey =
```

Comment:
At the moment these are there just because we use them internally, but we have #27 for reviewing the examples before the release, plus #59 for the env vars. Both should help us to get rid of the comments! :pray: